### DISCUSSION TITLE
Not working? $CHANGE_LOGIN_PASSWORD = false;

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14692] [General] [answered]
It looks like changing below settings in Security.php doesn't bring expected effect.
At list in my 6.1.0 v
Anyone else?

/** Changing the settings by the user is possible true/false */
	public static $CHANGE_LOGIN_PASSWORD = false;

**Comment by vovpff** [comment, marked as answer]
@s2b-git this is fixed by me and will be available in 6.2+ update

Ref to https://github.com/YetiForceCompany/YetiForceCRM/pull/14143

**Comment by s2b-git** [comment]
Thank you
